metpy/plots/ctables.py

```diff
@@ -127,6 +129,13 @@ class ColortableRegistry(dict):
     matplotlib's Normalize instances to go with the colortable.
     """
 
+    def __getitem__(self, key):
+        """Handle viridis deprecation."""
+        if key == 'viridis':
+            warnings.warn('Viridis has been deprecated in v0.11. Please use '
+                          "matplotlib's 'viridis'.", metpyDeprecation)
+        return super().__getitem__(key)
```

Comment:
For Python 2.7 compatibility, this needs to be `super(ColortableRegistry, self)`.

Reply:
None of the ctables tests pass when updating the return line to your suggestion